toc/working-groups/docs.md

```diff
@@ -31,6 +31,10 @@ name: Documentation
 execution_leads:
 - name: Anita Flegg
   github: anita-flegg
+- name: Stuart Clements
+  github: stuclem
+- name: Richard Johnson
+  github: RichardJJG
```

Comment:
Hi @anita-flegg,

is the intention to make Stuart and Richard execution leads or approvers in the Docs WG? Currently, the pr adds them as execution leads. I just wanted to clarify because the title of the pr mentions the approver role.